webapp/src/client.ts

```diff
@@ -555,8 +566,21 @@ export default class CallsClient extends EventEmitter {
         };
 
         logDebug('adding stream to peer', screenStream.id);
+
+        // Always send a fallback track (VP8 encoded) for receivers that don't yet support AV1.
         this.peer.addStream(screenStream);
 
+        if (this.config.enableAV1 && this.av1Codec) {
+            if (this.config.simulcast) {
+                logWarn('both simulcast and av1 support are enabled');
```

Comment:
What about prioritizing AV1 over simulcast?
And either way, maybe we should throw a configuration error if the system admin selects both? Otherwise they will select them and then wonder why they're not seeing any improvement...

Reply:
@cpoile I honestly didn't want to overcomplicate it for some experimental features that we don't yet know if we want to retain. Our config can still be changed by different means (e.g., JSON file), so then we'd have to decide what to do in case the mutually exclusive settings are turned on (e.g. fail on start?).

The idea of logging the warning was that it would be trivial to determine if they fall into such a case. This is also documented in the config itself. I feel that may be enough for the time being.